editoast/src/schema/operation/update.rs

```diff
@@ -94,6 +95,13 @@ impl DataObject {
         if obj_railjson.get_id() != &update.obj_id {
             return Err(OperationError::ModifyId.into());
         }
+
+        if let RailjsonObject::SpeedSection { railjson } = &obj_railjson {
+            railjson
+                .validate()
+                .map_err(|err| OperationError::InvalidPatch(err.to_string()))?;
+        };
```

Comment:
Same here, we can implement `Validate` to the `RailjsonObject` struct and `Operation`.